WalletWasabi.Documentation/FAQ.md

```diff
@@ -0,0 +1,178 @@
+FAQ for [Wasabi Wallet](https://github.com/zkSNACKs/WalletWasabi)
+By [6102bitcoin](https://twitter.com/6102bitcoin)
+
+Note: Many of these Q&A have been copied from real users, see footer for acknowledgements. 
+
+# Index
+- Pre-Install
+- Install
+- Pre-Mix
+- Mixing
+- Post-Mix
+- Meta
+- Acknowledgements
+
+# Pre-Install
+
+### Who is behind Wasabi?
+
+The Company that is developing Wasabi is zkSNACKs LTD ([twitter](@Zksnacks_LTD) | [website](https://zksnacks.com/))  
+Members as of 2019.05.08:
+
+| Name              | Twitter                                                 | Position			   |
+| ------------------| --------------------------------------------------------|	-----------------------|
+| Gergely Hajdú 	| [@gergely_hajdu](https://twitter.com/gergely_hajdu) 	  | Co-CEO & Co-Founder	   |
+| Bálint Harmat		| [@bharmat84](https://twitter.com/bharmat84) 			  | Co-CEO & Co-Founder    |
+| Ádám Ficsór		| [@nopara73](https://twitter.com/nopara73)	              | Developer & Co-Founder |
+| Lucas Ontivero	| [@lontivero](https://twitter.com/)                      | Developer & Co-Founder |
+| Dan Walmsley   	| ([@danwalmsley](https://github.com/danwalmsley))        | Developer              |
+| Dávid Molnár   	| ([@molnardavid84](https://twitter.com/molnardavid84))   | Developer              |
+
+# Install
+
+### How do I install Wasabi?
+
+Follow [this guide](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/Guides/InstallInstructions.md)
+
+If you use Windows there is a [video guide](https://youtu.be/D8U53PFEsVk).
+
+### Do I need to run Tor?
+
+All Wasabi network traffic goes via Tor by default - no need to set up Tor yourself. If you do already have Tor, and it is running, then Wasabi will try to use that first.  
```

Comment:
```suggestion
All Wasabi network traffic goes via Tor by default - no need to set up Tor yourself. If you do already have Tor, and it is running, then Wasabi will try to use that first.  
You can turn off Tor in the Settings. Note that in this case you are still private, except when you coinjoin and when you broacast a transaction. In the first case, the coordinator would learn the links between your inputs and outputs based on your IP address, in the second case, if you happen to broadcast a transaction of yours to a full node that is spying on you, it will learn the link between your transaction and your IP address.
```